packages/frontend/src/hooks/toaster/useToaster.tsx

```diff
@@ -90,10 +90,6 @@ const useToaster = () => {
                             )}
                         </Stack>
                     ) : undefined,
-                onOpen: (props: NotificationProps) => {
-                    rest.onOpen?.(props);
-                    if (props.id) openedKeys.current.add(props.id);
-                },
```

Comment:
raised some warnings in the console. mantine does not have that.  + we don't use this anywhere in the codebase